tools/self-check

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+USER='testu'
+PASS='testp'
+
+DEBTEST=`lsb_release -a 2> /dev/null | grep Distributor | awk '{print $3}'`
+
+if [[ "$DEBTEST" == "Ubuntu" ]]; then
+  TYPE="debs"
+  PYTHONPACK="/usr/lib/python2.7/dist-packages"
+elif [[ -f "/etc/redhat-release" ]]; then
+  TYPE="rpms"
+  PYTHONPACK="/usr/lib/python2.7/site-packages"
+else
+  echo "Unknown Operating System"
+  exit 2
+fi
+
+INSTALL=`${PYTHONPACK}/st2common/bin/st2-setup-tests`
+
+EXITCODE=$?
+echo $INSTALL
+
+INSTALL=`${PYTHONPACK}/st2common/bin/st2-setup-examples`
+
+EXITCODE=$?
+echo $INSTALL
+
+ST2_AUTH_TOKEN=`st2 auth ${USER} -p ${PASS} | grep token | awk '{ print $4 }'`
+export ST2_AUTH_TOKEN=${ST2_AUTH_TOKEN}
+
+TEST_ACTION_LIST=`st2 action list --pack=tests | awk '{ print $2 }' | grep -v "|" | grep -v "ref"`
```

Comment:
I don't mind assuming the token exists in the env already.  That way I can just rip out the whole bit with the user/pass.  I'll keep the install bits in but wrap them in some logic to see if they are already installed.